### ISSUE TITLE
Change the type of content property in ToastMessage

### ISSUE THREAD

**Issue by prakash1998** [primefaces/primereact#1761]
```
[-] feature request
```
It will be good, if `message.content` property of ToastMessage component also support functional component along with ReactElement

currently it supports 
```
content : <div>component</div>
```

better if it can also support
```
content : ({ close }) => <div>component</div>
```
it will provide more flexibility when user need to render custom component inside toast

i would like to submit pull request if needed, please respond me if you like to have this feature

**Comment by prakash1998** [comment]
hey it will be helpful if you pass close as prop of functional component

**Comment by mertsincan** [comment]
Fixed in #1805. You can use onClose callback for this issue.
